src/map/battle.c

```diff
@@ -7092,7 +7092,11 @@ static const struct battle_data {
 	{ "pet_status_support",                 &battle_config.pet_status_support,              0,      0,      1,              },
 	{ "pet_attack_support",                 &battle_config.pet_attack_support,              0,      0,      1,              },
 	{ "pet_damage_support",                 &battle_config.pet_damage_support,              0,      0,      1,              },
+#ifdef RENEWAL
+	{ "pet_support_min_friendly_re",        &battle_config.pet_support_min_friendly,        910,    0,      950,            },
```

Comment:
just curios, why a new config? since both just using different value? 
could have just remain one config, its not possible to trigger/use both in server.
this config seem redundant.